src/core/core.layouts.js

```diff
@@ -57,13 +57,14 @@ function buildLayoutBoxes(boxes) {
 	var right = sortByWeight(filterByPosition(layoutBoxes, 'right'));
 	var top = sortByWeight(filterByPosition(layoutBoxes, 'top'), true);
 	var bottom = sortByWeight(filterByPosition(layoutBoxes, 'bottom'));
+	const centerHorizontal = filterByPosition(layoutBoxes, 'centerHorizontal');
 
 	return {
 		leftAndTop: left.concat(top),
-		rightAndBottom: right.concat(bottom),
+		rightAndBottom: right.concat(bottom).concat(centerHorizontal),
```

Comment:
Why in `rightAndBottom` and not in `vertical`?

Reply:
`centerHorizontal` is an X axis that is vertically centered. I'm open to other names though, since I can forsee it being confusing

Reply:
What about something like:
```js
scales: {
  my : {
    axis: 'x',
    position: 'center'
 }
} 
```
could also extend to `position : 'y=700' `

Reply:
sure, will see how that goes

Reply:
`position: 'center'` makes sense to me

Reply:
Cool, I just pushed some changes. Now you specify `position: 'center'` and `axis: 'x'` in the scale options to make it work. If we like where this is going, I can look into the y axis next

Reply:
The direction looks good to me!